src/containerapp/azext_containerapp/_validators.py

```diff
@@ -38,6 +39,15 @@ def _is_number(s):
         return False
 
 
+def validate_revision_suffix(value):
```

Comment:
@haroonf can you check this change. While I am happy that CXP is helping, but I would rather have us take care of this or at least review this. lets check with backend folks for a validation or regex they use on their end. Thanks!

Reply:
@RakeshMohanMSFT please remove code changes from this & let the PG team take care of this thanks!